manager/models.py

```diff
@@ -851,6 +851,7 @@ def tracking_urls(self):
                 urls.append(URL.objects.get_or_create(
                     instance = self,
                     name     = href,
```

Comment:
Right now we're just reusing the href as the name field, but in the future we could use the link text as the name.